src/app/Main/Body/QueryPage/Parameter/QueryPage-ParameterContent.component.tsx

```diff
@@ -45,12 +45,12 @@ export function QueryPageParameterContentComponent(): JSX.Element {
         }
     });
 
-    const AddParameterElement = ((filterAttributes: FilterType[], filterValues: FilterInterface): JSX.Element[] => {
+    const createParameterComponent = (
+        filterAttributes: FilterType[],
+        filterValues: FilterInterface
+    ): JSX.Element[] => {
         const elements: JSX.Element[] = [];
-        pairwise(filterAttributes, function func(
-            current: FilterType,
-            next: FilterType
-        ): void {
+        pairwise(filterAttributes, (current, next): void => {
```

Comment:
You can remove the "void" too.
To explain: If you declare a function you type it, if you use a function or define it like here, typescript will infer the types for you. If you are interested you can google function declaration vs function definition.
Another thought to it: Would you use something like: foo('xyz': string) instead of foo('xyz') ? foo already says it gets a string, so why typing it twice? It's the same like foo((x:string):void=>{}) vs foo((x)=>{})
Actually type inference is more complicated than i stated it here but it is only interesting for the guys who write the compilers (in most cases).

Reply:
I went with your idea of letting the browser decide how many elements are displaied in a row. 
Nevertheless, thanks for the input. I'll try to keep it in mind next time.